features/app/components/AbstractApp.js

```diff
@@ -22,6 +31,30 @@ export class AbstractApp extends Component {
     }
 
     /**
+     * Init lib-jitsi-meet and create local participant when component is going
+     * to be mounted.
+     *
+     * @inheritdoc
+     */
+    componentWillMount() {
```

Comment:
And, actually, it seems it's not recommended way to use ```dispatch``` in constructor according to Redux author comments in https://github.com/reactjs/react-redux/issues/129. So I'm going to move ```dispatch(roomSet(room))``` to ```componentWillMount``` as well.